python/data_extraction/content_export.py

```diff
@@ -46,14 +46,17 @@ def get_content(base_path,
                 content_store_url=plek.find('draft-content-store')):
     content_dict = __get_content_dict(base_path, content_store_url)
 
+    if not content_dict:
```

Comment:
Don't think these changes are necessary because empty dictionaries evaluate to false anyway

Reply:
Yeah, I was just trying to make the "skip" return type more explicit, and different from a dictionary.